app/serializers/subject_set_serializer.rb

```diff
@@ -2,9 +2,14 @@ class SubjectSetSerializer
   include RestPack::Serializer
   include FilterHasMany
 
-  attributes :id, :display_name, :set_member_subjects_count, :metadata,
-    :created_at, :updated_at, :href
+  attributes :id, :display_name, :set_member_subjects_count,
+    :retired_set_member_subjects_count, :metadata, :created_at, :updated_at,
+    :href
   can_include :project, :workflows
   can_sort_by :display_name
   can_filter_by :display_name
+
+  def retired_set_member_subjects_count
+    SubjectWorkflowCount.by_set(@model.id).retired.count
```

Comment:
yeah - could be in the case of workflow subjects in more than 1 set. I can't see an easy way around that edge case..?